src/generic/stage2/tape_writer.h

```diff
@@ -75,6 +78,10 @@ simdjson_inline void tape_writer::append_u64(uint64_t value) noexcept {
   next_tape_loc++;
 }
 
+simdjson_inline void tape_writer::append_bigint(const uint8_t *const src, const uint8_t *const end) noexcept {
+  append(end - src, internal::tape_type::BIGINT);
```

Comment:
Where is the bigint written? Note that the DOM tape is meant to be complete, it does not include a reference to the original document. You could write it to the string buffer, but as far as I can tell, it is not done now.

Reply:
Thanks! This morning I also saw that this part was incorrect.